Pascal/Pascal.sublime-syntax

```diff
@@ -12,17 +12,69 @@ file_extensions:
 
 first_line_match: |-
   (?xi:
-    ^ \s* // .*? -\*- .*? \bpas(cal)?\b .*? -\*-  # editorconfig
+    ^ \s* // .*? -\*- .*? {{first_line_language}} .*? -\*-                      # editorconfig
```

Comment:
The `{{first_line_language}}` makes sense only, if ...

1. a corresponding rule to highlight the language name exists in one of the (comment-) contexts to keep in sync with.
2. this pattern was assigned to all syntaxes like "shebang".

Not sure it's worth it, though.